cluster/container.go

```diff
@@ -10,3 +16,50 @@ type Container struct {
 	Info   dockerclient.ContainerInfo
 	Engine *Engine
 }
+
+// Status returns a human-readable description of the state
+// Stoken from docker/docker/daemon/state.go
+func (c *Container) Status() string {
+	s := c.Info.State
+	if s.Running {
+		if s.Paused {
```

Comment:
Should we reuse `startedAt`.
`startedAt := units.HumanDuration(time.Now().UTC().Sub(s.StartedAt))`
All next three statements just need to use it.